.coveragerc

```diff
@@ -776,6 +776,9 @@ omit =
     homeassistant/components/rest/switch.py
     homeassistant/components/ring/camera.py
     homeassistant/components/ripple/sensor.py
+    homeassistant/components/rituals_perfume_genie/switch.py
+    homeassistant/components/rituals_perfume_genie/__init__.py
+    homeassistant/components/rituals_perfume_genie/const.py
```

Comment:
```suggestion
```

Reply:
It was not clear whether const.py should be included in the list or not. This file is not explicitly tested and other integrations also listed it.
`pytest --cov=homeassistant/components/rituals_perfume_genie/ --cov-report term-missing -- tests/components/rituals_perfume_genie/test_*.py` returns `100%` coverage for `const.py`.